### ISSUE TITLE
Fill in specific build instructions for MacOS

### ISSUE THREAD

**Issue by jkfindeisen** [gpufit/GPUfit#25]
after having it built on a MacOS machine.

**Comment by jkfindeisen** [comment]
Björn asked the IT service and they do not have a Mac with CUDA GPU. I can actually live very well with us not testing how GPUfits builds on Mac. We can argue that it should work in general but we give no guarantee.

**Comment by superchromix** [comment]
https://aws.amazon.com/ec2/instance-types/

**Comment by jkfindeisen** [comment]
Good idea. However, searched for MacOS on the page. Didn't find anything.

**Comment by jkfindeisen** [comment]
A friend of Björn might have a Mac with a CUDA GPU. 
Holger Starks department (http://www.mpibpc.mpg.de/stark) might have Macs with CUDA GPUs.

**Comment by jkfindeisen** [comment]
Unclear if this is needed before submission?
